src/Abp/Application/Features/FeatureManager.cs

```diff
@@ -43,11 +55,19 @@ public Feature Get(string name)
             return feature;
         }
 
+        /// <summary>
+        /// Gets all the features
+        /// </summary>
         public IReadOnlyList<Feature> GetAll()
         {
             return Features.Values.ToImmutableList();
         }
 
+        /// <summary>
+        /// ??? Not sure if this is the best description
+        /// Creates a disposable <see cref="FeatureProvider"/> from the given provider type
```

Comment:
Not sure about this description

Reply:
We don't add summary for private methods normally. So, ignore it :)